lib/eblox/data/post.ex

```diff
@@ -6,12 +6,11 @@ defmodule Eblox.Data.Post do
   require Logger
 
   @fsm """
-  idle --> |read| read
-  idle --> |delete| deleted
-  read --> |parse| parsed
-  read --> |parse| errored
-  read --> |delete| deleted
+  idle --> |read!| read
+  read --> |parse!| parsed
+  read --> |parse!| errored
```

Comment:
Like where? There is a single transition (well, single event, to be precise,) that might result (end up) in two different states. So it’ll be entered automatically because the bang _and_ `on_transition/4` handler would be responsible for routing to either one or another state.

Reply:
Ok, I misunderstood. Or this explanation is confusing. Two destination states, two possible transitions to these states but only one event. A transition is a change from one state to another, so in this case there are two possible transitions.

Reply:
Yeah, I am onto writing this down to docs.

Reply:
https://github.com/am-kantox/finitomata/issues/23